Code/Framework/AzToolsFramework/AzToolsFramework/ViewportSelection/EditorTransformComponentSelection.cpp

```diff
@@ -1594,57 +1579,62 @@ namespace AzToolsFramework
             }
 
-            // note: must use sorted entityIds based on hierarchy order when updating transforms
-            for (AZ::EntityId entityId : sharedScaleState->m_entityIds)
+            const auto sumVectorElements = [](const AZ::Vector3& vec)
             {
-                auto entityIdLookupIt = m_entityIdManipulators.m_lookups.find(entityId);
-                if (entityIdLookupIt == m_entityIdManipulators.m_lookups.end())
-                {
-                    continue;
-                }
+                return vec.GetX() + vec.GetY() + vec.GetZ();
+            };
 
-                const AZ::Transform initial = entityIdLookupIt->second.m_initial;
-                const float initialScale = initial.GetUniformScale();
+            const float uniformScale =
+                action.m_start.m_sign * sumVectorElements(action.LocalScaleOffset() - sharedScaleState->m_savedScale);
 
-                const auto sumVectorElements = [](const AZ::Vector3& vec)
+            if (action.m_modifiers.Ctrl())
+            {
+                // adjust the manipulator scale when ctrl is held
+                SetManipulatorViewBaseScale(AZ::GetClamp(
+                    sharedScaleState->m_initialViewScale + uniformScale, MinManipulatorViewBaseScale, MaxManipulatorViewBaseScale));
+            }
+            else
+            {
+                // note: must use sorted entityIds based on hierarchy order when updating transforms
+                for (AZ::EntityId entityId : sharedScaleState->m_entityIds)
                 {
-                    return vec.GetX() + vec.GetY() + vec.GetZ();
-                };
+                    auto entityIdLookupIt = m_entityIdManipulators.m_lookups.find(entityId);
+                    if (entityIdLookupIt == m_entityIdManipulators.m_lookups.end())
+                    {
+                        continue;
+                    }
 
-                const float uniformScale =
-                    action.m_start.m_sign * sumVectorElements(action.LocalScaleOffset() - sharedScaleState->m_savedScaleOffset);
-                const float scale = AZ::GetClamp(1.0f + uniformScale / initialScale, AZ::MinTransformScale, AZ::MaxTransformScale);
-                const AZ::Transform scaleTransform = AZ::Transform::CreateUniformScale(scale);
+                    const AZ::Transform initial = entityIdLookupIt->second.m_initial;
+                    const float initialScale = initial.GetUniformScale();
 
-                switch (InfluenceFromModifiers(action.m_modifiers))
-                {
-                case Influence::Individual:
-                    {
-                        const AZ::Transform pivotTransform = TransformNormalizedScale(entityIdLookupIt->second.m_initial);
-                        const AZ::Transform transformInPivotSpace = pivotTransform.GetInverse() * initial;
+                    const float scale = AZ::GetClamp(1.0f + uniformScale / initialScale, AZ::MinTransformScale, AZ::MaxTransformScale);
```

Comment:
Not sure about adding the scale here vs multiplying. I notice in the video when you move the mouse the manipulator does not stay aligned with the mouse position. Not necessarily something to look at in this PR though.

Reply:
Hmm I think I made this mistake before 😅 Let me test again and see, it did seem to be doing what I expected but I'll double check when I get a moment

Reply:
Hmm I looked at this again and this part didn't really change (other than indentation), I do think this is the correct behavior as we expect today (maybe we could discuss more offline to clarify), but I think for this change this is okay to leave for now